includes/class-migration.php

```diff
@@ -262,6 +262,31 @@ private static function migrate_from_2_6_0() {
 		self::update_options_key( 'activitypub_blog_user_identifier', 'activitypub_blog_identifier' );
 	}
 
+	/**
+	 * Update actor-mode settings.
+	 */
+	private static function migrate_from_4_0_0() {
+		$blog_profile    = \get_option( 'activitypub_enable_blog_user', false );
+		$author_profiles = \get_option( 'activitypub_enable_users', false );
+
```

Comment:
Not sure what needs changing but when I upgrade on a site that had both User and Blog profiles enabled, nothing is set in the `activitypub_actor_mode` option, but the settings page looks like "Author Only" is selected. This persists even if I press "Save" on that page, still no `activitypub_actor_mode` setting

Reply:
I guess that can happen when `$blog_profile` and `$author_profiles` are not `'1'`. There's no `else` clause as a fallback below that would account for that.

A unit test might be helpful for this function to cover these scenarios?

Reply:
oops, the migration is never called 🫣

I messed that up, because I thought I can merge #894 first (and that has all the 4.0.0 migration stuff)

Reply:
I also added some unit tests